### ISSUE TITLE
api.dashboard.nebra.com

### ISSUE THREAD

**Issue by whittinghamj** [NebraLtd/dashboard-issues#16]
I would love to access some of the OTA details that the Nebra hotspots post to your dashboard.

This information would be the cpu / mem / lan or wifi traffic usage etc stats.

Either a RESTful endpoint of a post forward per hotspot and I can set my own forwarding enpoint.

Thanks

**Comment by vpetersson** [comment]
@whittinghamj Adding is an API is on our roadmap. I think we'll aim for feature parity to the dashboard, but this is TBD at this point in time.

**Comment by jsonpoindexter** [comment]
@vpetersson with the announcement of the dashboard becoming a paid service is there still plans to implement a consumable API for owners?

**Comment by vpetersson** [comment]
> @vpetersson with the announcement of the dashboard becoming a paid service is there still plans to implement a consumable API for owners?

Yes. There will be an API for the dashboard provided to users of the dashboard. Are there any particular features you are looking for in the API?
